flycheck.el

```diff
@@ -6747,7 +6747,7 @@ See URL `http://golang.org/cmd/go'."
   ;; `temporary-file-name'.
   ;; TODO: Switch to `null-device'` when < Go 1.6 support is removed.
   ;; See: https://github.com/flycheck/flycheck/issues/838
-  :command ("go" "test" "-c" "-o" temporary-file-name)
+  :command ("go" "test" (option-flag "-i" flycheck-go-build-install-deps) "-c" "-o" temporary-file-name)
```

Comment:
Please wrap this line, it's too long. Put the `(option-flag …)` part on a separate line, and the arguments after it on another separator line.